build/asset-manifest.json

```diff
@@ -4,7 +4,7 @@
     "main.js": "/static/js/main.js",
     "main.js.map": "/static/js/main.js.map",
     "index.html": "/index.html",
-    "precache-manifest.8b635f0101f1f7d77058e7739a638d8e.js": "/precache-manifest.8b635f0101f1f7d77058e7739a638d8e.js",
+    "precache-manifest.c1ae79e9493f8affecc6e07bab3ee72b.js": "/precache-manifest.c1ae79e9493f8affecc6e07bab3ee72b.js",
```

Comment:
I see that the old `precache-manifest.*.js` file was removed, but the new one hasn’t been added to this PR—should it be?

Reply:
So, I don't think that the precache manifest is in any way useful, it's just one part of the mountain of junk that create-react-app generates. However, I've now pushed it into the branch!